core/server/api_container/server/service_network/default_service_network.go

```diff
@@ -109,195 +89,22 @@ type DefaultServiceNetwork struct {
 func NewDefaultServiceNetwork(
 	enclaveUuid enclave.EnclaveUUID,
 	apiContainerInfo *ApiContainerInfo,
-	isPartitioningEnabled bool,
 	kurtosisBackend backend_interface.KurtosisBackend,
 	enclaveDataDir *enclave_data_directory.EnclaveDataDirectory,
-	networkingSidecarManager networking_sidecar.NetworkingSidecarManager,
-	enclaveDb *enclave_db.EnclaveDB,
+	enclaveDb *enclave_db.EnclaveDB, //TODO we are going to use it soon, for the APIC restart project
```

Comment:
the enclave DB is not used now, but I leave it because I think we will need it for APIC restart